api.planx.uk/modules/flows/validate/service/index.ts

```diff
@@ -49,29 +49,30 @@ const validateAndDiffFlow = async (
     ...flattenedFlow[key],
   }));
 
-  const validationChecks = [];
-  const sections = validateSections(flattenedFlow);
-  const inviteToPay = validateInviteToPay(flattenedFlow);
-  const fileTypes = validateFileTypes(flattenedFlow);
-  const projectTypes = validateProjectTypes(flattenedFlow);
-  validationChecks.push(sections, inviteToPay, fileTypes, projectTypes);
+  // const validationChecks = [];
+  // const sections = validateSections(flattenedFlow);
+  // const inviteToPay = validateInviteToPay(flattenedFlow);
+  // const fileTypes = validateFileTypes(flattenedFlow);
+  // const projectTypes = validateProjectTypes(flattenedFlow);
+  // validationChecks.push(sections, inviteToPay, fileTypes, projectTypes);
 
-  // Arrange list of validation checks in order of status: Fail, Warn, Pass, Not applicable
-  const failingChecks = validationChecks.filter((v) => v.status == "Fail");
-  const warningChecks = validationChecks.filter((v) => v.status === "Warn");
-  const passingChecks = validationChecks.filter((v) => v.status === "Pass");
-  const notApplicableChecks = validationChecks.filter(
-    (v) => v.status === "Not applicable",
-  );
-  const sortedValidationChecks = failingChecks
-    .concat(warningChecks)
-    .concat(passingChecks)
-    .concat(notApplicableChecks);
+  // // Arrange list of validation checks in order of status: Fail, Warn, Pass, Not applicable
+  // const failingChecks = validationChecks.filter((v) => v.status == "Fail");
+  // const warningChecks = validationChecks.filter((v) => v.status === "Warn");
+  // const passingChecks = validationChecks.filter((v) => v.status === "Pass");
+  // const notApplicableChecks = validationChecks.filter(
+  //   (v) => v.status === "Not applicable",
+  // );
+  // const sortedValidationChecks = failingChecks
+  //   .concat(warningChecks)
+  //   .concat(passingChecks)
+  //   .concat(notApplicableChecks);
 
   return {
     alteredNodes,
     message: "Changes queued to publish",
-    validationChecks: sortedValidationChecks,
+    // validationChecks: sortedValidationChecks,
+    validationChecks: undefined,
```

Comment:
I think this needs to `[]` for the following line to not throw on error (`Cannot read properties of undefined (reading 'filter')`) - 

https://github.com/theopensystemslab/planx-new/blob/21755b044fa6d94c68933dc790ef3044b9ebe4f4/editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Publish/PublishFlowButton.tsx#L180